project/MiMa.scala

```diff
@@ -27,10 +27,11 @@ object MiMa extends AutoPlugin {
   val currentFork = "10.1."
 
   override val projectSettings = Seq(
-    mimaPreviousArtifacts :=
+    mimaPreviousArtifacts := {
```

Comment:
Looks good. Should we create a ticket to verify these settings once we release the second version for 2.13?

We could also just start to add versions here just to see if it already catches something?

Reply:
We had 10.1.7 in the list earlier, but I moved it out. The reason for that was that our new approach with the varargs was not binary compatible with our previous approach for the 2.13 artifacts. Adding exclusions for those would be possible, but I was afraid they would obscure possible 'real' problems on 2.11 and 2.12.

Reply:
Yep, agreed.